src/types.ts

```diff
@@ -0,0 +1,151 @@
+  | 'conference-booths-large'
+  | 'conference-booths-mixed'
+  | 'conference-booths-small'
+  | 'conference-booths-unfurnished'
+  | 'conference-breakout'
+  | 'conference-breakout-unfurnished'
+  | 'conference-deluxe-beach'
+  | 'conference-deluxe-booths-mixed-1'
+  | 'conference-deluxe-booths-mixed-2'
+  | 'conference-deluxe-keynote-east'
+  | 'conference-deluxe-keynote-west'
+  | 'conference-deluxe-lounge'
+  | 'conference-deluxe-main'
+  | 'conference-deluxe-posters-24-1'
+  | 'conference-deluxe-posters-24-2'
+  | 'conference-foyer'
+  | 'conference-foyer-unfurnished'
+  | 'conference-keynote'
+  | 'conference-keynote-unfurnished'
+  | 'conference-main-hall'
+  | 'conference-main-hall-unfurnished'
+  | 'conference-main-large'
+  | 'conference-poster-room-12'
+  | 'conference-poster-room-12-unfurnished'
+  | 'conference-poster-room-24'
+  | 'poster room with 24 posters. movable doors.'
+  | 'custom-entrance'
+  | 'cyberpunk-apartment'
+  | 'detective-office'
+  | 'diner-medium'
+  | 'diner-small'
+  | 'dive-bar'
+  | 'education-breakout-small'
+  | 'education-classroom-large'
+  | 'education-classroom-small'
+  | 'education-common-room-unfurnished'
+  | 'education-hs-common-room'
+  | 'education-lecture-hall'
+  | 'education-recess'
+  | 'education-recess-unfurnished'
+  | 'education-room-unfurnished'
+  | 'empty-100x100'
+  | 'empty-22x22'
+  | 'empty-45x45'
+  | 'empty-room-medium-brick'
+  | 'empty-room-small-brick'
+  | 'event-entrance-foyer'
+  | 'event-entrance-main'
+  | 'family-gathering'
+  | 'family-gathering-cozy'
+  | 'family-gathering-lny'
+  | 'family-gathering-nye-2021'
+  | 'game-room-large'
+  | 'game-room-medium'
+  | 'game-room-small'
+  | 'games-pool'
+  | 'games-track'
+  | 'greenhouse'
+  | 'keynote'
+  | 'keynote-outdoor'
+  | 'lounge-large'
+  | 'moonpond_t'
+  | 'nautical-bar-exterior'
+  | 'nautical-bar-lower'
+  | 'nautical-bar-upper'
+  | 'night-market'
+  | 'night-tent-a'
+  | 'night-tent-b'
+  | 'office-cozy'
+  | 'office-fancy'
+  | 'office-forest'
+  | 'office-inc'
+  | 'office-industrial'
+  | 'office-large-dark-all-hands'
+  | 'office-large-dark-main'
+  | 'office-large-wood-all-hands'
+  | 'office-medium-dark'
+  | 'office-medium-light'
+  | 'office-medium-wood'
+  | 'office-modern'
+  | 'office-pirate-above'
+  | 'office-pirate-bellow'
+  | 'office-pirate-cabin'
+  | 'office-small-dark'
+  | 'office-small-light'
+  | 'office-small-wood'
+  | 'office-space-medium'
+  | 'office-space-small'
+  | 'office-t'
+  | 'office-tiny-dark'
+  | 'office-tiny-light'
+  | 'office-tiny-wood'
+  | 'office-unfurnished'
+  | 'officeMedium'
+  | 'open-mic'
+  | 'open-mic-pride'
+  | 'park-day'
+  | 'park-night'
+  | 'park-t'
+  | 'racetrack'
+  | 'recess'
+  | 'rooftop'
+  | 'space-station'
+  | 'speakeasy'
+  | 'speed-gathering'
+  | 'study'
+  | 'tavern-main'
+  | 'tavern-upper'
+  | 'tetris-tournament'
+  | 'tiny-island'
+  | 'uni-lab'
+  | 'uni-large-lab'
+  | 'uni-lec'
+  | 'uni-rec'
+  | 'waterfront'
+  | 'zen-garden';
```

Comment:
I'm curious where did you get these?

Reply:
from gather.town itself when you are creating a space then clicking the choose from template will show all these

Reply:
All good this to me 😎